org.eclipse.acute/src/org/eclipse/acute/dotnetnew/DotnetNewWizard.java

```diff
@@ -56,55 +60,84 @@ public boolean performFinish() {
 		}
 
 		if (!template.isEmpty()) {
```

Comment:
With dotnet-core 1.1 `dotnet new` does generate a C# project and no templates are available. We need this to work with the wizard.